config/200-clusterrole.yaml

```diff
@@ -24,6 +24,9 @@ rules:
   - apiGroups: [""]
     resources: ["configmaps", "services", "secrets", "events", "serviceaccounts"]
     verbs: ["get", "list", "create", "update", "delete", "patch", "watch"]
+  - apiGroups: [""]
+    resources: ["namespaces"]
```

Comment:
you can just add this to the entry above 🤷

Reply:
don't want to create/delete rights.